torch_sparse/narrow.py

```diff
@@ -1,11 +1,69 @@
 from typing import Tuple
 
+import torch
 from torch_sparse.storage import SparseStorage
 from torch_sparse.tensor import SparseTensor
 
 
 def narrow(src: SparseTensor, dim: int, start: int,
```

Comment:
Can we add a test for this?

Reply:
If I get what you mean, I will refactor the proposed code, adding a new function/method `narrow_block`. 
For that function/method I will add a test.